core/src/main/java/org/jruby/RubyBignum.java

```diff
@@ -236,6 +236,54 @@ public static BigInteger long2big(long arg) {
      *  ================
      */
 
+    /** rb_big_digits
+     *
+     */
+    @Override
+    public RubyArray digits(ThreadContext context, IRubyObject base) {
+        BigInteger self = (this).getValue();
```

Comment:
You could just access the field here, or at least call it without the parens. I assume this is just a mistake converting the RubyFixnum-only logic.